tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1379,6 +1379,11 @@ def test_copy_template_file(ansible_zos_module, encoding):
             src=temp_template,
             dest=dest_path,
             use_template=True,
+            template_parameters={
+                # This should make sure that we don't get stuff like '&#39' when
+                # rendering quotes.
+                "autoescape": False
```

Comment:
Since we are adding a functionality that changes the file contents (escaping) we should also validate the file contents afterwards, for both escaping and not escaping.